src/shared/copy/notifications.ts

```diff
@@ -1468,6 +1468,12 @@ export const testNotificationFailure = (
   message: `Failed to send the test alert to ${source}. Please try again`,
 })
 
+export const exportAlertToTaskSuccess = (): Notification => ({
+  ...defaultSuccessNotification,
+  message: `An alert task has been successfully created.
+  Please go to your tasks, and confirm the task runs correctly.`,
```

Comment:
@juliajames12   let me know what's the preferred message for here. Thanks!

Reply:
An alternative idea:
(1) "Export to task" is disabled until the user tests the task.
(2) then afterwards...can export to task.
(3) the success message can then be simplified. Since will also be a functional/correct task.

This will then fully close issue #3875.

Reply:
Spoke with @juliajames12 .
We made a decision to:
* have the "test alert" and "export alert to task" to remain wholly separate.
* have a double API call made when exporting alert to task. The first API call will try running the query & surfacing any errors, and the second API call will export the task (auto success create) & return the UX notification on export.